config/routes.rb

```diff
@@ -41,6 +41,8 @@
   resources :editions do
     collection do
       get 'existing_work'
+      get "import"
```

Comment:
Why are you supporting a get here? I think you should only support the post

Reply:
The `get` is to show the page to import. In other words, to show the form that will receive the store's URL